spec/index.bs

```diff
@@ -1417,6 +1417,22 @@ applications should generally request the "worst" limits that work for their con
         allowed in {{GPUPipelineLayoutDescriptor/bindGroupLayouts}}
         when creating a {{GPUPipelineLayout}}.
 
+    <tr><td><dfn>maxBindingsPerBindGroup</dfn>
+        <td>{{GPUSize32}} <td>[=limit class/maximum=] <td>640
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed number of {{GPUBindGroupLayoutEntry}}.{{GPUBindGroupLayoutEntry/binding}}
+        values when creating a {{GPUBindGroupLayout}}.
+
+        The [=supported limit=] for an [=adapter=] must be &gt;
```

Comment:
I think still should be "&ge;"

Also the rename caused me to think a little about the "per shader stage" vs "per bind group" difference, and I think we actually need to guarantee this is &ge; `sum(...) * (max number of stages = 2)`?

Reply:
Done. (Thought I pushed that last Thursday, but forgot that I held off because of some bikeshed errors I was getting at the time.)